src/measure_theory/integral/interval_average.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import analysis.convex.integral
+import measure_theory.integral.interval_integral
+
+/-!
+# Integral average over an interval
+
+In this file we prove formulas for the averate of a function over an interval w.r.t. the Lebesgue
+measure:
+
```

Comment:
Can you also say close to the beginning of the file-level docstring that you define `⨍ x in a..b` in this file, and mention its definition (because otherwise it is not possible to understand the meaning of the theorems)?